src/webextension/list/manage/components/account-summary.css

```diff
@@ -124,18 +145,20 @@
   height: 16px;
   background-repeat: no-repeat;
   background-size: 16px 16px;
-  margin-inline-end: 1ch;
+  margin-right: 8px;
   vertical-align: middle;
+  margin-top: -2px;
```

Comment:
This seems a bit brittle to me, but I'm not sure if there's a better way to make it look "right"...

Reply:
@jimporter I agree, wasn't able to get them to line up perfectly unless the text was within a span. Or could use position relative to move this up but figured it wasn't much difference. Let me know if you want it updated to another way.